website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -161,24 +200,33 @@ In addition to this, the dynamodb table and bucket version need to created as we
 :::
 
-For the `Deploy Dev Cumulus and ORCA Stack`  stage, add the following variables. 
+After hitting the play button on `Deploy Dev Cumulus and ORCA Stack`, but before hitting `Run` in the popup, replace the following variables with yours.
 
-- RDS_SECURITY_GROUP
-- RDS_USER_ACCESS_SECRET_ARN
-- DB_HOST_ENDPOINT
+- CUMULUS_AWS_ACCESS_KEY_ID
+- CUMULUS_AWS_SECRET_ACCESS_KEY
+- PREFIX
+- AWS_ACCOUNT_ID (for cumulus sandbox account)
+- AWS_SUBNET_ID1
+- AWS_SUBNET_ID2
+- VPC_ID
 - EARTHDATA_CLIENT_ID
 - EARTHDATA_CLIENT_PASSWORD
-- CUMULUS_ORCA_DEPLOY_TEMPLATE_VERSION
+- DB_ADMIN_PASSWORD
+- DB_USER_PASSWORD
+- DB_HOST_ENDPOINT
+- RDS_SECURITY_GROUP
+- RDS_USER_ACCESS_SECRET_ARN
+- CUMULUS_ORCA_DEPLOY_TEMPLATE_VERSION (optional)
 
-The RDS variables `RDS_SECURITY_GROUP`, `RDS_USER_ACCESS_SECRET_ARN` and `DB_HOST_ENDPOINT` can be found from output logs of the previous `Deploy Dev RDS Stack` stage. Note that a new earthdata application will need to be first created if using a new prefix for new deployment which will give the values for `EARTHDATA_CLIENT_ID` and `EARTHDATA_CLIENT_PASSWORD`. `CUMULUS_ORCA_DEPLOY_TEMPLATE_VERSION` is the branch you want to check out in the [deployment repo](https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-deploy-template/browse) such as `v11.1.1-v4.0.1`.
+The RDS variables `RDS_SECURITY_GROUP`, `RDS_USER_ACCESS_SECRET_ARN` and `DB_HOST_ENDPOINT` can be found from output logs of the previous `Deploy Dev RDS Stack` stage as `security_group_id`, `admin_db_login_secret_arn`, and `rds_endpoint` respectively. Note that a new earthdata application will need to be first created if using a new prefix for new deployment which will give the values for `EARTHDATA_CLIENT_ID` and `EARTHDATA_CLIENT_PASSWORD`. `CUMULUS_ORCA_DEPLOY_TEMPLATE_VERSION` is the branch you want to check out in the [deployment repo](https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-deploy-template/browse) such as `v11.1.1-v4.0.1`.
 
-Note that `RDS_USER_ACCESS_SECRET_ARN` value from the initial run should be recorded, as they may be hidden on future deployments. In addition, the jobs may need to be run multiple times to get past deployment errors if there is one. If an error is raised saying `Cloudwatch log groups already exist`, then manually delete all the cloudwatch log groups and corresponding lambdas having the same name as the log groups from the AWS console and retry running the job.
\ No newline at end of file
+Note that `admin_db_login_secret_arn` value from the initial run should be recorded, as they may be hidden on future deployments. In addition, the jobs may need to be run multiple times to get past deployment errors if there is one. If an error is raised saying `Cloudwatch log groups already exist`, then manually delete all the cloudwatch log groups and corresponding lambdas having the same name as the log groups from the AWS console and retry running the job.
```

Comment:
the instructions will change once ORCA-464 is done since user will no longer need to manually input those RDS and security group variables. Just a heads up